pynocular/patch_models.py

```diff
@@ -319,12 +369,35 @@ def _evaluate_binary_expression(
     if operator == is_:
         operator = lambda x, y: x is y
 
+    # The sqlalchemy `is_not` operator does not work on evaluated columns, so we replace
+    # it with the standard `!=` operator.
+    if operator == is_not:
+        operator = lambda x, y: x != y
```

Comment:
pynocular in general doesn't support `is_not` in its interface. How are you planning to use this?

Reply:
I was using it in one of the `where_expressions` in a custom filter

Reply:
oh okay, cool